examples/acquiring_data/querying_and_loading_SHARP_data.py

```diff
@@ -0,0 +1,44 @@
+"""
+===============================
+Querying and loading SHARP data
+===============================
+
+In this example we will demonstrate how to acquire [Spaceweather HMI Active Region Patch (SHARP)](http://jsoc.stanford.edu/doc/data/hmi/sharp/sharp.htm) data and load it into a `sunpy.map.Map`.
+"""
+import os
+
+import astropy.units as u
+
+import sunpy.map
+from sunpy.net import Fido
+from sunpy.net import attrs as a
+
+###################################################################
+# To search for SHARP data, we will need to query the [JSOC/Stanford](http://jsoc.stanford.edu/).
+# We will use `Fido <sunpy.net.fido_factory.Fido>`
```

Comment:
This in fact did not work: `/home/runner/work/sunpy/sunpy/docs/generated/gallery/acquiring_data/querying_and_loading_SHARP_data.rst:42: WARNING: py:obj reference target not found: sunpy.net.fido_factory.Fido`

Reply:
hi, i'm unaware how one checks this. I could revert it back to `sunpy.net.fido_factory.UnifiedDownloaderFactory` which was working before but would like to check if `sunpt.net.Fido` would work, could you point me to what documentation will be helpful here?

Reply:
Go back to `sunpy.net.fido_factory.UnifiedDownloaderFactory` for now.

Reply:
sure, thanks!